app/furniture/marketplace/stripe_accounts_controller.rb

```diff
@@ -1,5 +1,5 @@
 class Marketplace
-  class StripeAccountsController < FurnitureController
+  class StripeAccountsController < Controller
     def create
       # TODO: Should we be using https://stripe.com/docs/connect/oauth-express-accounts instead?!
```

Comment:
Not from this PR, but we probably can remove this TODO for now.

Reply:
True